utils/virtcontainers-setup.sh

```diff
@@ -16,4 +16,62 @@
 
 #!/bin/bash
 
-bash $GOPATH/src/github.com/containers/virtcontainers/ci/ci-setup.sh
+if [[ $EUID -ne 0 ]]; then
+	echo "This script must be run as root"
+	exit 1
+fi
+
+echo "Create temporary build directory /tmp/virtcontainers/build"
```

Comment:
There is a *lot* or repetition in this file. Could you add some variables to clean it up a bit? This sort of thing:

```
tmpdir=$(mktemp)
echo "blah blah $tmpdir"
rm -rf "$tmpdir"
```

Note the unique temporary directory. Can we switch to that (safer). If not, I think this hard-coded path deserves a comment, particularly as it would be useful to be able to run these scripts on a test system.